src/Hyperion/Main.hs

```diff
@@ -78,7 +81,8 @@ options = do
          (Options.strOption
             (Options.long "output" <>
              Options.short 'o' <>
-             Options.metavar "FILENAME"))
+             Options.help "Outputs the benchark to the directory" <>
```

Comment:
Change to "Where to write the benchmarks output. Can be a directory name." Options don't act.